### DISCUSSION TITLE
POSTMAN for exams

### DISCUSSION THREAD

**Discussion by Brilbroeder** [SafeExamBrowser/seb-win-refactoring#543] [Q&A]
Hi,
for dev-students we use postman as an essential tool when developing rest-api's. Unfortunatly at the moment POSTMAN does not to seem to be running together with SafeExamBrowser.

It starts but it gets killed. Can I enabled extended logging to see why SEB kills POSTMAN, hence what extra processes to allow in the config ?
Has any already managed to have POSTMAN being available to students through SEB ?

Any suggestions ?

**Comment by dbuechel** [comment]
This information is in the application log of the client component, to be found under `%LocalAppData%\SafeExamBrowser\Logs`. Or you can view the log in real time by allowing access to the application log in your configuration: https://safeexambrowser.org/windows/win_usermanual_en.html#SecurityPane.
